rgw/v2/tests/s3_swift/test_dynamic_bucket_resharding.py

```diff
@@ -174,6 +174,76 @@ def test_exec(config, ssh_con):
                 log.info("Expected number of shards created")
         else:
             raise TestExecError("Expected number of shards not created")
+
+    if config.disable_dynamic_shard:
+        log.info("Testing disable of DBR")
+        bucket_stat_cmd = f"radosgw-admin bucket stats --bucket {bucket.name}"
+        op1 = utils.exec_shell_cmd(bucket_stat_cmd)
+        json_doc = json.loads(op1)
+        num_objects = json_doc["usage"]["rgw.main"]["num_objects"]
+        num_shards_created = json_doc["num_shards"]
+        if num_shards_created > 11:
+            log.info(f"Dynamic Re-sharding is successfull!")
+        else:
+            raise AssertionError("Dynamic Re-sharding FAILED!")
+
+        log.info("Disabling resharding in Zonegroup")
+
+        zonegroup_get_cmd = "radosgw-admin zonegroup get"
+        out = utils.exec_shell_cmd(zonegroup_get_cmd)
+        zonegroups = json.loads(out)
+        zonegroup = zonegroups.get("name")
```

Comment:
We can use it as 
zonegroups = json.loads(utils.exec_shell_cmd(zonegroup_get_cmd))